### ISSUE TITLE
Configure dative.ca subdomains to resolve to INT

### ISSUE THREAD

**Issue by jrwdunham** [dativebase/dativebaseclj#28]
A/C:

- With the help of INT staff, configure dative.ca subdomains to resolve to INT server.
  - app.dative.ca => INT server 132.229.188.226 (Dative app & OLD API)
  - www.dative.ca => INT server 132.229.188.226 (Dative website)
  - www.old.dative.ca => INT server 132.229.188.226 (OLD website)
  - onlinelinguisticdatabase.org is no longer needed.

BLOCKED by #30
